diff --git a/lib/browser/prompttemplate-contribution.js b/lib/browser/prompttemplate-contribution.js
index 5817edcaa24b9c8daa4854cd5a07db89ac1cb885..e954cfff85c1aa320a6c4ba2888351b31f84ac57 100644
--- a/lib/browser/prompttemplate-contribution.js
+++ b/lib/browser/prompttemplate-contribution.js
@@ -60,7 +60,7 @@ let PromptTemplateContribution = class PromptTemplateContribution {
         monaco.languages.register({
             id: PROMPT_TEMPLATE_LANGUAGE_ID,
             'aliases': [
-                'AI Prompt Template'
+                core_1.nls.localize('theia/ai/core/promptTemplate/language', 'AI Prompt Template')
             ],
             'extensions': [
                 exports.PROMPT_TEMPLATE_EXTENSION,
diff --git a/src/browser/prompttemplate-contribution.ts b/src/browser/prompttemplate-contribution.ts
index 625bb21f243e7b297c9a7935242476f52157e65b..174c02dd04e9f02cc183badd479bf735fd03bebc 100644
--- a/src/browser/prompttemplate-contribution.ts
+++ b/src/browser/prompttemplate-contribution.ts
@@ -76,7 +76,7 @@ export class PromptTemplateContribution implements LanguageGrammarDefinitionCont
         monaco.languages.register({
             id: PROMPT_TEMPLATE_LANGUAGE_ID,
             'aliases': [
-                'AI Prompt Template'
+                nls.localize('theia/ai/core/promptTemplate/language', 'AI Prompt Template')
             ],
             'extensions': [
                 PROMPT_TEMPLATE_EXTENSION,
